src/app/legacy/containers/Metadata/index.test.jsx

```diff
@@ -525,6 +535,28 @@ it('should render the twitter metatags', async () => {
   });
 });
 
+it('should render the twitter handle of the author', async () => {
```

Comment:
Do you think its worth having another test here to check the `twitter:creator` for CPS pages to ensure they aren't affected by this change?

Reply:
Thanks for the feedback Aaron.
It certainly is. Could you confirm that you're referring to adding assertions,
that check that the `twitter:creator` behaviour is specific to `src/app/pages/ArticlePage`
and it doesn't apply to page types set in [src/app/routes/cpsAsset/index.jsx#L25-L31](https://github.com/bbc/simorgh/blob/fbea8683459b6095dfd7e5a1c0a6315bb529712c/src/app/routes/cpsAsset/index.jsx#L25-L31) below?
```
STY, CSP, PGL, MAP and FIX
```
Proceeding to look into how to add these assertions.

Reply:
Yea i think its `STY, PGL, MAP` pages that use the `<CpsMetadata />` component. This component uses the base `<Metadata />` component which the `<ArticleMetadata />` component also uses as its base.

Its just to ensure that `STY, PGL, MAP` pages behave as they did before this change, as this change is for Optimo articles only.

Reply:
Thanks for the pointer.
[Here](https://github.com/bbc/simorgh/pull/10401/files#diff-e46197c977470f13da2ae8f337aabb88f5f2827e8e5fa843e90a90b00f8bec92R563-R661), I've added assertions to test the `twitter:creator` inference for the Front Page,
Story Page (STY), Media Asset Page (MAP) and Photo Gallery (PGL).

Reply:
Looks great, thanks!